File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int ans =0;
        for(int i=0; i<nums.size();i++){
            int t=0;
            while(nums[i]!=0){
                int l = nums[i]%10;
                t++;
                nums[i] = nums[i]/10;
            }
            if(t%2==0)ans++;
        }
        return ans;
    }
};

File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        ans = 0
        for n in nums:
            c =0 
            while n != 0:
                n //= 10
                c += 1
            if c % 2 == 0:
                ans += 1
        
        return ans